src/pages/ClassComponent.mdx

```diff
@@ -176,3 +176,92 @@ export class ChildClassComponentWithObjectPropsMemoized extends Component<
 but when parent component changes `obj.str` re-rendering happens, as expected:
 
 <ObjectPropsClassComponent isMemoized={true} changeProps={true} />
+
+---
+
+### Regular Component - Method Component
```

Comment:
Do people really call it method components?

Reply:
I'm not sure if it has any official name. If yes, we can use it, if not, this term explains what it is about

Reply:
Yeah, I just invented it 😄 Will create an article in Wiki